packages/svelte/src/compiler/phases/2-analyze/css/Selector.js

```diff
@@ -830,18 +845,62 @@ class Block {
 	}
 }
 
-/** @param {import('#compiler').Css.Selector} selector */
-function group_selectors(selector) {
+const InvisibleCombinator = { type: /** @type {"Combinator"} **/ ("Combinator"), name: ' ', start: -1, end: -1}
```

Comment:
not sure what the type annotation is doing here, it seems a bit redundant. if we need an annotation to type `InvisibleCombinator` as a combinator (and maybe we don't?) then we can do it like this:

```suggestion
/** @type {import('#compiler').Css.Combinator} */
const InvisibleCombinator = { type: 'Combinator', name: ' ', start: -1, end: -1}
```